Detectors/TRD/workflow/include/TRDWorkflow/VdAndExBCalibSpec.h

```diff
@@ -76,11 +88,20 @@ class VdAndExBCalibDevice : public o2::framework::Task
     LOG(info) << "Finalizing calibration";
     mCalibrator->checkSlotsToFinalize(o2::calibration::INFINITE_TF);
     sendOutput(ec.outputs());
+    if (mOutFile && mOutTree) {
+      LOG(info) << "Writing calibration to file";
+      mOutFile->cd();
+      mOutTree->Write();
```

Comment:
I am not sure if there is a better way to do this...

Reply:
similar here just a call to `mCalibrator->closeOutputFile()`. Besides that there is nothing wrong with the way you are writing